src/base/cvar.ml

```diff
@@ -27,17 +32,75 @@ let to_constant_and_terms ~equal ~add ~mul ~zero ~one =
     (c, ts)
 
 module Unsafe = struct
+  (** Directly converts a number into a variable.
+      This operation is considered unsafe as you would normally increment a counter to obtain a variable that doesn't collide with another one.
+    *)
   let of_index v = Var v
 end
 
+(***********************
+ *  SIGS FOR FUNCTORS  *
+ ***********************)
+
+module type cvar_module_type = sig
```

Comment:
Nit: module types normally have a leading uppercase letter (and in fact I'm surprised that the parser accepted this!)

Reply:
interesting, looks like the manual uses all uppercases: https://v2.ocaml.org/manual/moduleexamples.html